lib/diagnostic.rb

```diff
@@ -108,10 +108,10 @@
 # Use shorthand to make this one line long.
 # Write your response as a Ruby string.
 
-Diagnostic.person_hash = 'your answer here'
+Diagnostic.person_hash = 'person[:favorite_number] = 14'
```

Comment:
First we need to make the Hash, it would look like this:

`'person = {first_name: "Lauren", favorite_number: 7}'`